app/src/lib/editors/linux.ts

```diff
@@ -128,6 +128,27 @@ const editors: ILinuxExternalEditor[] = [
       '.local/share/JetBrains/Toolbox/scripts/goland',
     ],
   },
+  {
+    name: 'JetBrains CLion',
+    paths: [
+      '/snap/bin/clion',
+      '.local/share/JetBrains/Toolbox/scripts/clion1',
+    ],
+  },
+  {
+    name: 'JetBrains Rider',
+    paths: [
+      '/snap/bin/rider',
+      '.local/share/JetBrains/Toolbox/scripts/rider',
+    ],
+  },
```

Comment:
Linter is failing on this:

```suggestion
  {
    name: 'JetBrains CLion',
    paths: ['/snap/bin/clion', '.local/share/JetBrains/Toolbox/scripts/clion1'],
  },
  {
    name: 'JetBrains Rider',
    paths: ['/snap/bin/rider', '.local/share/JetBrains/Toolbox/scripts/rider'],
  },
```